PR: https://github.com/odoo/odoo/pull/99411

From: 2bc141e6ba38149b65f0062a742746c783c6dd4d
From: svs-odoo
Date: 2022-09-12 11:49:15

Structural Changes: 3
Total Changes: 13

[REF] purchase(_stock): Removes price diff account

- Removes related fields in views and models.

Part-of: odoo/odoo#99411

================================= pseudo patch: =================================

--- a/addons/purchase/models/product.py
+++ b/addons/purchase/models/product.py
@@ -11,10 +11,6 @@ class ProductTemplate(models.Model):
     _name = 'product.template'
     _inherit = 'product.template'
 
-    property_account_creditor_price_difference = fields.Many2one(
-        'account.account', string="Price Difference Account", company_dependent=True,
-        help="This account is used in automated inventory valuation to "\
-             "record the price difference between a purchase order and its related vendor bill when validating this vendor bill.")
     purchased_product_qty = fields.Float(compute='_compute_purchased_product_qty', string='Purchased', digits='Product Unit of Measure')
     purchase_method = fields.Selection([
         ('purchase', 'On ordered quantities'),
@@ -80,15 +76,6 @@ class ProductProduct(models.Model):
         return action
 
 
-class ProductCategory(models.Model):
-    _inherit = "product.category"
-
-    property_account_creditor_price_difference_categ = fields.Many2one(
-        'account.account', string="Price Difference Account",
-        company_dependent=True,
-        help="This account will be used to value price difference between purchase price and accounting cost.")
-
-
 class ProductSupplierinfo(models.Model):
     _inherit = "product.supplierinfo"
 
